frontend/src/metabase/nav/components/ProfileLink.jsx

```diff
@@ -106,11 +105,7 @@ function ProfileLink({ user, adminItems, onLogout }) {
         items={generateOptionsForUser()}
         triggerIcon="gear"
         triggerProps={{
-          color: color("text-medium"),
-          hover: {
-            backgroundColor: color("brand"),
-            color: color("text-white"),
-          },
+          isInvertedColor: true,
```

Comment:
Combine multiple explicit props into a single variant so it's easier to implement the styled component.